modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+[id="odc-creating-a-binding-connection-between-components_{context}"]
+== Creating a binding connection between components
+
+You can establish a binding connection with Operator-backed components.
+
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
+The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
```

Comment:
```suggestion
This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. 

To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator. The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.

When you create the service binding, the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.

.Binding connector
image::odc-binding-connector.png[]
```

Reply:
It will be covered as part of https://issues.redhat.com/browse/RHDEVDOCS-4234